### PR TITLE
image-create: increase boot timeout on Arch Linux

### PR DESCRIPTION
**Pull request by jelly**
The one time `reflector-init` service blocks sshd from starting and can take quite a while to complete as it sequentially checks every mirrors repository database.

 * [x] image-refresh arch

### CODE REVIEW

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/arch-20221212-124141.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/arch-20221212-132626.log

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/arch-20221212-134949.log

**Comment by cockpituous** [comment]
image-refresh arch done: https://github.com/cockpit-project/bots/commits/image-refresh-arch-20221212-155520

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/arch-20221212-155555.log

**Comment by cockpituous** [comment]
image-refresh arch done: https://github.com/cockpit-project/bots/commits/image-refresh-arch-20221212-173527

**Comment by martinpitt** [comment]
Thanks @jelly ! Please push the new image here and let the tests run -- still too early to review

File: image-create [hunk 1]
```diff
@@ -79,7 +79,13 @@ class MachineBuilder:
         """Prepare a test image further by running some commands in it."""
         self.machine.start()
         try:
-            self.machine.wait_boot(timeout_sec=120)
+            timeout = 120
+            # except with arch, the `reflector-init` service blocks ssh and it takes quite a while to
```
Comment (martinpitt) [inline image-create:83]: Please add a `HACK:` prefix here, to include that in greps.

**Review by martinpitt — CHANGES_REQUESTED** [review]

**Comment by martinpitt** [comment]
Ugh, this is a sea of red and "Unable to reach machine arch-127.0.0.2-2501 via ssh: 127.0.0.2:2501".

**Comment by jelly** [comment]
Pulled the image down, ran ./test/image-prepare arch but `./test/verify/check-connection -vst TestConnection.testSocket` works fine here.

**Comment by jelly** [comment]
The new arch image no longer uses reflector so this is unrequired.

**Pull request closed without merging**